sites/mosul/configs/openmrs/initializer_config/drugs/drugs.csv

```diff
@@ -297,7 +291,7 @@
 "8101ead5-0c6d-40e5-a7f3-eb69d69b2212","NIFEDIPINE, 10mg, immediate-release tab.","35a9ad57-bdd1-4f43-b355-d22bdfbd3250","81d0f25d-4721-40cb-94ff-99601a260b28","10mg"
 "0269a9c2-1000-47d2-ba2b-a5f5553e1a82","NIFURTIMOX, 120mg, tab.","892604bb-0989-421d-8df0-76866079bfd9","c1f4a088-32a8-4b34-8162-944bb6ed026f","120mg"
 "b1ebb531-ead3-4744-af95-6609ea5d0bca","NITROFURANTOIN, 100mg, caps.","e46d140d-ba91-4234-b0cc-8f33ef714ad1","c2c7e944-e9d6-4b2d-bf1e-b542dbdbb634","100mg"
-"d6658c05-d03b-407c-8f33-f67eb2edd9ad","NYSTATIN, 100,000 IU/ml, oral susp. + graduated dropper","1806171f-7506-4418-b3f2-25290ff86a14","fe152bf2-f35e-41fa-bc78-bbb94b60d6d2","100,000 IU/ml"
+"d6658c05-d03b-407c-8f33-f67eb2edd9ad","NYSTATIN, 100,000 IU/ml, oral susp. + graduated dropper","80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA","fe152bf2-f35e-41fa-bc78-bbb94b60d6d2","100,000 IU/ml"

```

Comment:
why are u shifting this to the CIEL concept?

Reply:
Reverted the change